src/modules/events/EventHeader.tsx

```diff
@@ -2,24 +2,25 @@ import React from "react";
 import Image from "next/image";
 import Event from "./EventType";
 
-type EventHeaderProps = Pick<Event, "title" | "image">;
+type EventHeaderProps = {
+  event: Event;
+};
 
-const EventHeader = ({ title, image }: EventHeaderProps) => (
-  <div className="relative h-64 w-full overflow-hidden rounded-lg shadow-sm sm:w-64">
-    {image && (
-      <div className="relative h-[65%] w-full">
+const EventHeader = ({ event }: EventHeaderProps) => {
```

Comment:
I think  passing `image, title` should be fine here as whole event is not needed.